exporter/containerimage/exptypes/types.go

```diff
@@ -24,3 +25,19 @@ type Platform struct {
 	ID       string
 	Platform ocispecs.Platform
 }
+
+type BuildInfo struct {
+	Type  BuildInfoType `json:"type,omitempty"`
+	Ref   string        `json:"ref,omitempty"`
+	Alias string        `json:"alias,omitempty"`
```

Comment:
`Alias` is only used to match with the actual source ref because frontend might have already transformed a string user typed before generating LLB:

```json
{
   "type": "image",
   "ref": "docker.io/library/alpine:3.13",
   "alias": "docker.io/library/alpine:3.13@sha256:",
   "pin": "sha256:",
}
```

I will add some documentation around these keys.